lib/decanter/extensions.rb

```diff
@@ -36,12 +35,28 @@ def decant_create!(args, **options)
       end
 
       def decant(args, options={})
+        is_collection?(args, options[:is_collection]) ? decant_collection(args, options) : decant_args(args, options)
+      end
+
+      def decant_collection(args, options)
+        args.map { |resource| decant_args(resource, options) }
+      end
+
+      def decant_args(args, options)
         if specified_decanter = options[:decanter]
           Decanter.decanter_from(specified_decanter)
         else
           Decanter.decanter_for(self)
         end.decant(args)
       end
+
+      private
+
+      ## leveraging the approach used in the [fast JSON API gem](https://github.com/Netflix/fast_jsonapi#collection-serialization)
```

Comment:
Is two hashes to start a comment indicative of something?

Reply:
nope I think it was a js feels